docs/contributors/develop.md

```diff
@@ -0,0 +1,9 @@
+# Developer Contributions
+
+Please see [CONTRIBUTING.md](https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md) for technical details on how to setup and make contributions to the Gutenberg repository.
+
+* [Coding Guidelines](/docs/contributors/coding-guidelines.md) outline additional patterns and conventions used in the Gutenberg project.
```

Comment:
Can we add a paragraph above this?

> The following resources offer additional information for developers who wish to contribute to Gutenberg:

(Something like that… I'm not totally thrilled but it's a start)

And we should also include a link to the documentation contribution doc.